### PR TITLE
[DM-655] Update the public doc with all the feature of Multiple LNS

### PR DESCRIPTION
**Pull request by erashwinigayad**

### CODE REVIEW

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md [hunk 1]
```diff
@@ -7,13 +7,29 @@ layout: redirect
...
+This warning message shows up if a provided baseUrl is invalid.
 
-<img src="/images/device-protocols/lora-actility/lora-connectivity-invalid-credentials.png" alt="Account credentials" style="max-width: 100%">
+<img src="/images/device-protocols/lora-actility/lora-connectivity-invalid-url-error.png" alt="Account credentials" style="max-width: 100%">
 <br>
-To resolve this, provide correct credentials and try again.
+To resolve this, provide correct baseUrl and try again.
+
+<br>
+Below warning message shows up if a provided profile ID is invalid.
```
Comment (MWindel) [inline content/protocol-integration/lora-actility-bundle/troubleshooting.md:19]:
```suggestion
If the provided profile ID is invalid, a warning message shows up (see image below).
```

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md [hunk 2]
```diff
@@ -7,13 +7,29 @@ layout: redirect
...
+#### Authentication to the Actility platform failed.
 
-This warning message shows up if a provided profile ID, username or password is invalid.
+This warning message shows up if a provided baseUrl is invalid.
 
-<img src="/images/device-protocols/lora-actility/lora-connectivity-invalid-credentials.png" alt="Account credentials" style="max-width: 100%">
+<img src="/images/device-protocols/lora-actility/lora-connectivity-invalid-url-error.png" alt="Account credentials" style="max-width: 100%">
 <br>
-To resolve this, provide correct credentials and try again.
+To resolve this, provide correct baseUrl and try again.
```
Comment (MWindel) [inline content/protocol-integration/lora-actility-bundle/troubleshooting.md:16]:
```suggestion
To resolve this, provide the correct baseUrl and try again.
```

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md [hunk 3]
```diff
@@ -7,13 +7,29 @@ layout: redirect
 <a name="lora-connectivity-troubleshooting"></a>
 ### Connectivity
 
-#### Authorization to the LoRa platform failed
+#### Authentication to the Actility platform failed.
 
-This warning message shows up if a provided profile ID, username or password is invalid.
+This warning message shows up if a provided baseUrl is invalid.
```
Comment (MWindel) [inline content/protocol-integration/lora-actility-bundle/troubleshooting.md:12]:
```suggestion
This warning message shows up if the provided baseUrl is invalid.
```

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md [hunk 4]
```diff
@@ -7,13 +7,29 @@ layout: redirect
...
 <br>
-To resolve this, provide correct credentials and try again.
+To resolve this, provide correct baseUrl and try again.
+
+<br>
+Below warning message shows up if a provided profile ID is invalid.
+
+<img src="/images/device-protocols/lora-actility/lora-connectivity-invalid-profileId-error.png" alt="Account credentials" style="max-width: 100%">
+<br>
+To resolve this, provide correct profile ID and try again.
```
Comment (MWindel) [inline content/protocol-integration/lora-actility-bundle/troubleshooting.md:23]:
```suggestion
To resolve this, provide the correct profile ID and try again.
```

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md [hunk 5]
```diff
@@ -7,13 +7,29 @@ layout: redirect
...
+
+<br>
+Below warning message shows up if a provided username or password is invalid.
```
Comment (MWindel) [inline content/protocol-integration/lora-actility-bundle/troubleshooting.md:26]:
```suggestion
If the provided username or password is invalid, a warning message shows up (see image below).
```

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md [hunk 6]
```diff
@@ -7,13 +7,29 @@ layout: redirect
...
+
+<img src="/images/device-protocols/lora-actility/lora-connectivity-invalid-username-or-password-error.png" alt="Account credentials" style="max-width: 100%">
+<br>
+To resolve this, provide correct username and password and try again.
```
Comment (MWindel) [inline content/protocol-integration/lora-actility-bundle/troubleshooting.md:30]:
```suggestion
To resolve this, provide the correct username and password and try again.
```

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md [hunk 7]
```diff
@@ -67,10 +67,33 @@ To add another connection, click **Add Connection** and follow the steps above.
...
 If there are devices associated with the connection, an error message will appear, stating "Can not update the LNS Connection with `<name of LNS Connection>` as it's associated with `<number of devices>`. Visit the following URL to download the file: `/service/<agent-context-path>/lns-connection/<lns-connection-name>/device`".
+![Delete connection information](/images/device-protocols/sigfox/sigfox-admin-settings-delete.png)
+
+#### Authentication to the Sigfox platform failed.
+
+This warning message shows up if a provided baseUrl is invalid.
+
+![Invalid baseurl error](/images/device-protocols/sigfox/sigfox-connectivity-invalid-url-error.png)
+<br>
+To resolve this, provide correct baseUrl and try again.
```
Comment (MWindel) [inline content/protocol-integration/sigfox-bundle/connectivity-sigfox.md:85]:
```suggestion
To resolve this, provide the correct baseUrl and try again.
```

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md [hunk 8]
```diff
@@ -67,10 +67,33 @@ To add another connection, click **Add Connection** and follow the steps above.
...
+
+<br>
+Below warning message shows up if a provided parent group ID is invalid.
```
Comment (MWindel) [inline content/protocol-integration/sigfox-bundle/connectivity-sigfox.md:88]:
```suggestion
If the provided parent group ID is invalid, a warning message shows up (see image below).
```

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md [hunk 9]
```diff
@@ -67,10 +67,33 @@ To add another connection, click **Add Connection** and follow the steps above.
...
+
+![Invalid parent group ID error](/images/device-protocols/sigfox/sigfox-connectivity-invalid-groupId-error.png)
+<br>
+To resolve this, provide correct profile ID and try again.
```
Comment (MWindel) [inline content/protocol-integration/sigfox-bundle/connectivity-sigfox.md:92]:
```suggestion
To resolve this, provide the correct profile ID and try again.
```

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md [hunk 10]
```diff
@@ -67,10 +67,33 @@ To add another connection, click **Add Connection** and follow the steps above.
...
+
+<br>
+Below warning message shows up if a provided username or password is invalid.
```
Comment (MWindel) [inline content/protocol-integration/sigfox-bundle/connectivity-sigfox.md:95]:
```suggestion
If the provided username or password is invalid, a warning message shows up (see image below).
```

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md [hunk 11]
```diff
@@ -67,10 +67,33 @@ To add another connection, click **Add Connection** and follow the steps above.
...
+
+![Invalid credentials error](/images/device-protocols/sigfox/sigfox-connectivity-invalid-username-or-password-error.png)
+<br>
+To resolve this, provide correct username and password and try again.
```
Comment (MWindel) [inline content/protocol-integration/sigfox-bundle/connectivity-sigfox.md:99]:
```suggestion
To resolve this, provide the correct username and password and try again.
```

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 12]
```diff
@@ -41,7 +41,7 @@ To add another connection, click **Add Connection** and follow the steps above.
 Select the connection to be updated, make your edits, and save the connection.
 
 
-If there are devices associated with the connection, an error message will appear, stating "Can not update the LNS Connection with `<name of LNS Connection>` as it's associated with `<number of devices>`. Visit the following URL to download the file: `/service/<agent-context-path>/lns-connection/<lns-connection-name>/device`".
+If there are devices associated with the connection, an error message will appear, stating "Can not update the LNS Connection with `<name of LNS Connection>` as it's associated with `<number of devices>`. Click to download link with affected devices: `/service/<agent-context-path>/lns-connection/<lns-connection-name>/device`".
```
Comment (BeateRixen) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:44]: Click to download link with affected devices: > 
This sounds weird, I download a file, not a link?
Perhaps:
Click the link to download the file with the affected devices?

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 13]
```diff
@@ -50,4 +50,6 @@ If there are devices associated with the connection, an error message will appea
 
 Select the connection to be deleted and click **Delete**.
 
-If there are devices associated with the connection, an error message will appear, stating "Can not update the LNS Connection with `<name of LNS Connection>` as it's associated with `<number of devices>`. Visit the following URL to download the file: `/service/<agent-context-path>/lns-connection/<lns-connection-name>/device`".
+If there are devices associated with the connection, an error message will appear, stating "Can not update the LNS Connection with `<name of LNS Connection>` as it's associated with `<number of devices>`. Click to download link with affected devices: `/service/<agent-context-path>/lns-connection/<lns-connection-name>/device`".
```
Comment (BeateRixen) [inline content/protocol-integration/lora-actility-bundle/configure-credentials.md:53]: See above

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md [hunk 14]
```diff
@@ -67,10 +67,33 @@ To add another connection, click **Add Connection** and follow the steps above.
 Select the connection to be updated, make your edits, and save the connection.
 
 If there are devices associated with the connection, an error message will appear, stating "Can not update the LNS Connection with `<name of LNS Connection>` as it's associated with `<number of devices>`. Visit the following URL to download the file: `/service/<agent-context-path>/lns-connection/<lns-connection-name>/device`".
```
Comment (BeateRixen) [inline content/protocol-integration/sigfox-bundle/connectivity-sigfox.md:69]: Here this sentences stay unchanged? Visit the following URL to download the file:

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md [hunk 15]
```diff
@@ -67,10 +67,33 @@ To add another connection, click **Add Connection** and follow the steps above.
...
-
+![Update connection information](/images/device-protocols/sigfox/sigfox-connectivity-update.png)
 
 ###  To delete a connection
 
 Select the connection to be deleted and click **Delete**.
 
 If there are devices associated with the connection, an error message will appear, stating "Can not update the LNS Connection with `<name of LNS Connection>` as it's associated with `<number of devices>`. Visit the following URL to download the file: `/service/<agent-context-path>/lns-connection/<lns-connection-name>/device`".
```
Comment (BeateRixen) [inline content/protocol-integration/sigfox-bundle/connectivity-sigfox.md:76]: See above

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md [hunk 16]
```diff
@@ -67,10 +67,33 @@ To add another connection, click **Add Connection** and follow the steps above.
...
+![Delete connection information](/images/device-protocols/sigfox/sigfox-admin-settings-delete.png)
+
+#### Authentication to the Sigfox platform failed.
+
+This warning message shows up if a provided baseUrl is invalid.
+
+![Invalid baseurl error](/images/device-protocols/sigfox/sigfox-connectivity-invalid-url-error.png)
```
Comment (BeateRixen) [inline content/protocol-integration/sigfox-bundle/connectivity-sigfox.md:83]: Please do not use screenshots just to repeat the error message. Maintenance of screenshots causes a lot of work for you in the future. I would recommend to remove all screenshots in this section.

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md (hunk 5 above)
Comment (erashwinigayad) [reply content/protocol-integration/lora-actility-bundle/troubleshooting.md]: Corrected.

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md (hunk 8 above)
Comment (erashwinigayad) [reply content/protocol-integration/sigfox-bundle/connectivity-sigfox.md]: Corrected.

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md (hunk 12 above)
Comment (erashwinigayad) [reply content/protocol-integration/lora-actility-bundle/configure-credentials.md]: Corrected.

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md (hunk 13 above)
Comment (erashwinigayad) [reply content/protocol-integration/lora-actility-bundle/configure-credentials.md]: Corrected.

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md (hunk 14 above)
Comment (erashwinigayad) [reply content/protocol-integration/sigfox-bundle/connectivity-sigfox.md]: Corrected.

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md (hunk 15 above)
Comment (erashwinigayad) [reply content/protocol-integration/sigfox-bundle/connectivity-sigfox.md]: Corrected.

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md (hunk 16 above)
Comment (erashwinigayad) [reply content/protocol-integration/sigfox-bundle/connectivity-sigfox.md]: Corrected.

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md (hunk 8 above)
Comment (erashwinigayad) [reply content/protocol-integration/sigfox-bundle/connectivity-sigfox.md]: Corrected.

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md (hunk 10 above)
Comment (erashwinigayad) [reply content/protocol-integration/sigfox-bundle/connectivity-sigfox.md]: Removed this as it is suggested not to have many screenshots for error message.

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md [hunk 17]
```diff
@@ -7,13 +7,29 @@ layout: redirect
 <a name="lora-connectivity-troubleshooting"></a>
 ### Connectivity
 
-#### Authorization to the LoRa platform failed
+#### Authentication to the Actility platform failed.
```
Comment (BeateRixen) [inline content/protocol-integration/lora-actility-bundle/troubleshooting.md:10]:
```suggestion
#### Authentication to the Actility platform failed
```

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md (hunk 2 above)
Comment (BeateRixen) [inline content/protocol-integration/lora-actility-bundle/troubleshooting.md:16]:
```suggestion
To resolve this, provide a correct baseUrl and try again.
```

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md [hunk 18]
```diff
@@ -7,13 +7,29 @@ layout: redirect
...
 <br>
-To resolve this, provide correct credentials and try again.
+To resolve this, provide correct baseUrl and try again.
+
+<br>
+This warning message shows up if a provided profile ID is invalid.
+
+<img src="/images/device-protocols/lora-actility/lora-connectivity-invalid-profileId-error.png" alt="Account credentials" style="max-width: 100%">
+<br>
+To resolve this, provide correct profile ID and try again.
```
Comment (BeateRixen) [inline content/protocol-integration/lora-actility-bundle/troubleshooting.md:23]:
```suggestion
To resolve this, provide a correct profile ID and try again.
```

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md [hunk 19]
```diff
@@ -7,13 +7,29 @@ layout: redirect
...
+
+<br>
+This warning message shows up if a provided username or password is invalid.
+
+<img src="/images/device-protocols/lora-actility/lora-connectivity-invalid-username-or-password-error.png" alt="Account credentials" style="max-width: 100%">
+<br>
+To resolve this, provide correct username and password and try again.
```
Comment (BeateRixen) [inline content/protocol-integration/lora-actility-bundle/troubleshooting.md:30]:
```suggestion
To resolve this, provide a correct username and password and try again.
```

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md [hunk 20]
```diff
@@ -66,11 +66,18 @@ To add another connection, click **Add Connection** and follow the steps above.
...
 
 Select the connection to be deleted and click **Delete**.
 
-If there are devices associated with the connection, an error message will appear, stating "Can not update the LNS Connection with `<name of LNS Connection>` as it's associated with `<number of devices>`. Visit the following URL to download the file: `/service/<agent-context-path>/lns-connection/<lns-connection-name>/device`".
+If there are devices associated with the connection, an error message will appear, stating "Can not update the LNS Connection with `<name of LNS Connection>` as it's associated with `<number of devices>`. Click the link to download the file with the affected devices: `/service/<agent-context-path>/lns-connection/<lns-connection-name>/device`".
+![Delete connection information](/images/device-protocols/sigfox/sigfox-admin-settings-delete.png)
+
+#### Authentication to the Sigfox platform failed.
+
+Authentication to the Sigfox platform failed. Check if the Parent group Id and/or the credentials are correct.
```
Comment (BeateRixen) [inline content/protocol-integration/sigfox-bundle/connectivity-sigfox.md:81]:
```suggestion
Authentication to the Sigfox platform failed. Check if the Parent group ID and/or the credentials are correct.
```

File: content/protocol-integration/sigfox-bundle/connectivity-sigfox.md [hunk 21]
```diff
@@ -66,11 +66,18 @@ To add another connection, click **Add Connection** and follow the steps above.
...
+
+To resolve this, provide correct baseUrl or parent group ID or Username or password and try again.
```
Comment (BeateRixen) [inline content/protocol-integration/sigfox-bundle/connectivity-sigfox.md:83]:
```suggestion
To resolve this, provide a correct baseUrl or parent group ID or username or password and try again.
```

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md [hunk 22]
```diff
@@ -7,13 +7,29 @@ layout: redirect
...
-<img src="/images/device-protocols/lora-actility/lora-connectivity-invalid-credentials.png" alt="Account credentials" style="max-width: 100%">
+<img src="/images/device-protocols/lora-actility/lora-connectivity-invalid-url-error.png" alt="Account credentials" style="max-width: 100%">
 <br>
-To resolve this, provide correct credentials and try again.
+To resolve this, provide a correct baseUrl and try again.
+
+<br>
+This warning message shows up if a provided profile ID is invalid.
+
+<img src="/images/device-protocols/lora-actility/lora-connectivity-invalid-profileId-error.png" alt="Account credentials" style="max-width: 100%">
```
Comment (BeateRixen) [inline content/protocol-integration/lora-actility-bundle/troubleshooting.md:21]: remove and paraphrase the warning message

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md [hunk 23]
```diff
@@ -7,13 +7,29 @@ layout: redirect
...
+<br>
+To resolve this, provide a correct profile ID and try again.
+
+<br>
+This warning message shows up if a provided username or password is invalid.
+
+<img src="/images/device-protocols/lora-actility/lora-connectivity-invalid-username-or-password-error.png" alt="Account credentials" style="max-width: 100%">
```
Comment (BeateRixen) [inline content/protocol-integration/lora-actility-bundle/troubleshooting.md:28]: see above

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md [hunk 24]
```diff
@@ -7,13 +7,29 @@ layout: redirect
...
 ### Connectivity
 
-#### Authorization to the LoRa platform failed
+#### Authentication to the Actility platform failed
 
-This warning message shows up if a provided profile ID, username or password is invalid.
+This warning message shows up if a provided baseUrl is invalid.
 
-<img src="/images/device-protocols/lora-actility/lora-connectivity-invalid-credentials.png" alt="Account credentials" style="max-width: 100%">
+<img src="/images/device-protocols/lora-actility/lora-connectivity-invalid-url-error.png" alt="Account credentials" style="max-width: 100%">
```
Comment (BeateRixen) [inline content/protocol-integration/lora-actility-bundle/troubleshooting.md:14]: remove and paraphrase the warning message

Comment (erashwinigayad) [reply content/protocol-integration/lora-actility-bundle/troubleshooting.md]: Removed.

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md (hunk 23 above)
Comment (erashwinigayad) [reply content/protocol-integration/lora-actility-bundle/troubleshooting.md]: Removed.

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md (hunk 22 above)
Comment (erashwinigayad) [reply content/protocol-integration/lora-actility-bundle/troubleshooting.md]: Done.

File: content/protocol-integration/lora-actility-bundle/troubleshooting.md [hunk 25]
```diff
@@ -4,17 +4,6 @@ title: Troubleshooting
 layout: redirect
 ---
 
-<a name="lora-connectivity-troubleshooting"></a>
-### Connectivity
-
-#### Authorization to the LoRa platform failed
-
-This warning message shows up if a provided profile ID, username or password is invalid.
```
Comment (BeateRixen) [inline content/protocol-integration/lora-actility-bundle/troubleshooting.md:12]: You removed the entire troubleshooting hint now. Is that intentional?

Comment (erashwinigayad) [reply content/protocol-integration/lora-actility-bundle/troubleshooting.md:12]: No it was not intentional. I understood the comment wrongly and remove this section completely. corrected it.

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@erashwinigayad I have removed more images and re-structured the troubleshooting sections accordingly. From my side it is ok now.

**Comment by BeateRixen** [comment]
@erashwinigayad Please have one more approval from you team.

**Review by atulpandasag — APPROVED** [review]

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 26]
```diff
@@ -41,7 +41,7 @@ To add another connection, click **Add Connection** and follow the steps above.
 Select the connection to be updated, make your edits, and save the connection.
 
 
-If there are devices associated with the connection, an error message will appear, stating "Can not update the LNS Connection with `<name of LNS Connection>` as it's associated with `<number of devices>`. Visit the following URL to download the file: `/service/<agent-context-path>/lns-connection/<lns-connection-name>/device`".
+If there are devices associated with the connection, an error message will appear, stating "Can not update the LNS Connection with `<name of LNS Connection>` as it's associated with `<number of devices>`. Click the link to download the file with the affected devices: `/service/<agent-context-path>/lns-connection/<lns-connection-name>/device`".
```
[the remark this answers is not in the document]
Comment (erashwinigayad) [reply content/protocol-integration/lora-actility-bundle/configure-credentials.md]: corrected.

File: content/protocol-integration/lora-actility-bundle/configure-credentials.md [hunk 27]
```diff
@@ -50,4 +50,6 @@ If there are devices associated with the connection, an error message will appea
 
 Select the connection to be deleted and click **Delete**.
 
-If there are devices associated with the connection, an error message will appear, stating "Can not update the LNS Connection with `<name of LNS Connection>` as it's associated with `<number of devices>`. Visit the following URL to download the file: `/service/<agent-context-path>/lns-connection/<lns-connection-name>/device`".
+If there are devices associated with the connection, an error message will appear, stating "Can not update the LNS Connection with `<name of LNS Connection>` as it's associated with `<number of devices>`. Click the link to download the file with the affected devices:: `/service/<agent-context-path>/lns-connection/<lns-connection-name>/device`".
```
[the remark this answers is not in the document]
Comment (erashwinigayad) [reply content/protocol-integration/lora-actility-bundle/configure-credentials.md]: corrected.

File: content/protocol-integration/sigfox-bundle/sigfox-troubleshooting.md [hunk 28]
```diff
@@ -5,55 +5,53 @@ layout: redirect
...
 
 Active contracts with free slots are filtered based on the activation end time and tokens in use. Contracts in which the activation end time is higher than the current time or the activation end time is unlimited, and contracts in which the max tokens are higher than the tokens in use or the max tokens are unlimited will be considered.
 
 In order to resolve this error, please contact support.sigfox.com to create a contract for your Sigfox account.
 
-### Sigfox callbacks in backend.sigfox.com are not created correctly
+#### No Sigfox provider settings are found.
 
-![Callback information](/images/device-protocols/sigfox/sigfox-troubleshooting-callbacks.png)
+This warning message shows up when there are no credentials set up for the Sigfox account.
```
[the remark this answers is not in the document]
Comment (erashwinigayad) [reply content/protocol-integration/sigfox-bundle/sigfox-troubleshooting.md]: Done

**Pull request merged**